be/src/formats/parquet/column_chunk_reader.cpp

```diff
@@ -65,8 +34,13 @@ Status ColumnChunkReader::init(int chunk_size) {
     } else {
         start_offset = metadata().data_page_offset;
     }
-
-    _page_reader = std::make_unique<PageReader>(&_file, start_offset, metadata().total_compressed_size);
+    size_t size = metadata().total_compressed_size;
+    IBufferedInputStream* stream = _opts.sb_stream;
+    if (!_opts.use_sb_stream) {
```

Comment:
Is it needed to check null sb_stream here as the default use_sb_stream is true,  but sb_stream is null?

Reply:
No, `sb_stream` is hold by `GroupReader` always, so there is no possibility that it's null.